src/renderer/hooks/usePrevious.ts

```diff
@@ -0,0 +1,9 @@
+import { useEffect, useRef } from 'react'
+
+export default function usePrevious(value: any) {
```

Comment:
naming nitpick: isn't this more like useInitialValue?

Reply:
No, this is always giving the value which was set before the render cycle

Reply:
then it makes sense to document that in a comment to avoid future confusion